app/src/openapi_server/impl/jobs_api.py

```diff
@@ -0,0 +1,209 @@
+            raise HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=f"Existing job with ID '{job_id}' already exists",
+            )
+        return job
+
+    def dismiss(self, jobId: str) -> StatusInfo:
+        job_lock_key = f"job:{jobId}"
+        try:
+            with self.redis_locking_client.lock(job_lock_key):
+                job = self.check_job_integrity(jobId, new_job=False)
+                process_lock_key = f"process:{job.processID}"
+                with self.redis_locking_client.lock(process_lock_key):
+                    response = requests.delete(
+                        f"{self.settings.EMS_API_URL}/dags/{job.processID}/dagRuns/{job.jobID}",
+                        auth=self.ems_api_auth,
+                    )
+                    response.raise_for_status()
+                    crud.delete_job(self.db, job)
+                    dismissed_datetime = datetime.now()
+                    return StatusInfo(
+                        process_id=job.processID,
+                        type=job.type,
+                        job_id=job.jobID,
+                        status=StatusCode.DISMISSED,
+                        message="Job dismissed",
+                        updated=dismissed_datetime,
+                        created=job.created,
+                        started=job.started,
+                        finished=dismissed_datetime,
+                        progress=job.progress,
+                        links=job.links,
+                    )
+
+        except LockError:
+            raise HTTPException(
+                status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                detail="Unable to acquire lock. Please try again later.",
+            )
+        except requests.exceptions.HTTPError as e:
+            raise HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=f"Failed to delete DAG run {job.jobID} for DAG {job.processID}: {e}",
+            )
+        except Exception as e:
+            raise HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=str(e)
+            )
+
+    def get_jobs(self) -> JobList:
+        jobs = crud.get_jobs(self.db)
+        job_status_infos = [
+            StatusInfo(
+                process_id=job.processID,
+                type=job.type,
+                job_id=job.jobID,
+                status=job.status,
+                message=job.message,
+                updated=job.updated,
+                created=job.created,
+                started=job.started,
+                finished=job.finished,
+                progress=job.progress,
+                links=job.links,
+            )
+            for job in jobs
+        ]
+        return JobList(
+            jobs=job_status_infos,
+            links=[],
+        )
+
+    def get_result(self, jobId: str, prefer: str) -> Dict[str, InlineOrRefData]:
```

Comment:
i'm curious what this means in the airflow/cwl-dag context. With CWL, we can define the output of a workflow or CommandLinetool, and can have a lot of control over that w.r.t. application-packages and what _they_ should return. 

Thinking ahead, if we run stage_out, i wonder if it makes sense to publish the 'success_features.json' as the output for cwl_dag. This is future work, not something to address here, jsut more architecturally.